samples/gen-go-blog/server/router.go

```diff
@@ -59,28 +61,40 @@ func (s *Server) Serve() error {
 	ingestURL := os.Getenv("TRACING_INGEST_URL")
 	isLocal := os.Getenv("_IS_LOCAL") == "true"
 	if (tracingToken != "" && ingestURL != "") || isLocal {
-		// Add rate limited sampling. We will only sample [Param] requests per second
-		// and [MaxOperations] different endpoints. Any endpoint above the [MaxOperations]
-		// limit will be probabilistically sampled.
-		cfgSampler := &jaegercfg.SamplerConfig{
-			Type:          jaeger.SamplerTypeRateLimiting,
-			Param:         5,
-			MaxOperations: 100,
+		samplingRate := .01               // 1% of requests
```

Comment:
should we make this a `const`?

Reply:
it can be overrode by the env var, has to be a variable

Reply:
whoops meant it for the `lowerBoundRateLimiter`

Reply:
👍 
https://github.com/Clever/wag/pull/249/commits/80213552c3b630328bd2686d2dff05916cedf82a